RedHat-UX/red-hat-design-system
# feat(avatar): add rh-avatar

## What I did

1. Implement PFE 1:1 for rh-avatar (work in progress)


## Testing Instructions

1. [demo](https://deploy-preview-436--red-hat-design-system.netlify.app/components/avatar/demo/)
2. [context demo](https://deploy-preview-436--red-hat-design-system.netlify.app/components/avatar/demo/color-context/)

Component should use "block" (text below) layout at mobile screen sizes

## Notes to Reviewers

- the `pattern` attribute is not yet documented on ux.dot
